test/test_helper.rb

```diff
@@ -3,14 +3,18 @@
 require 'rails/test_help'
 require 'webmock'
 require 'vcr'
+require 'simplecov'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.